python/pyspark/sql/functions.py

```diff
@@ -48,6 +48,17 @@ def _(col):
     return _
 
 
+def _create_function(name, doc=""):
+    """ Create a column-based function by name"""
```

Comment:
`Running ./dev/run-tests and testing manually.` doesn't look testing it. Please test it manually at least.

Reply:
I did test it manually; isn't that what it says?